src/testsets.jl

```diff
@@ -245,13 +241,15 @@ function _flatten_results!(ts::AbstractTestSet)::Vector{<:AbstractTestSet}
     function inner!(childts::AbstractTestSet)
         # Make it a sibling
         update_testset_properties!(childts, ts)
-        childts.description = ts.description * "/" * childts.description
+        if depth > 0 || has_description(ts)
+            childts.description = ts.description * "/" * childts.description
+        end
```

Comment:
Currently we only skip appending unnamed top-level test sets from the nested testset description. I think it would be reasonable to skip any unnamed testset at any level but didn't want to work through all of the repercussions of such a change at this time.

Reply:
I agree i would want to think about that a little more